src/main/java/de/chojo/repbot/util/DiscordWebhookAppender.java

```diff
@@ -137,33 +138,40 @@ public void stop() {
 
     @Override
     public void append(LogEvent logEvent) {
+        var source = logEvent.getSource();
         // TODO: this shit seems to cause recursive logging calls
```

Comment:
I this todo still one or fixed by your PR as well?

Reply:
No its not fixed